src/solvers/dgsem_unstructured/containers_2d.jl

```diff
@@ -230,7 +230,6 @@ function init_interfaces!(interfaces, edge_information, boundary_names, n_elemen
 end
 
 
-# TODO: Clean-up meshes. Find a better name since it's also used for other meshes
```

Comment:
Was the naming issue resolved? I might've missed it.

Reply:
It's `UnstructuredBoundaryContainer2D` - it was specific to the `UnstructuredQuadMesh` (or however it was named before). Do you use the same stuff in 3D? If so, we should remove the `2D` part, I guess.

Reply:
I added it to the list in https://github.com/trixi-framework/Trixi.jl/issues/542

Reply:
Great, thanks!